src/pages/Start.vue

```diff
@@ -50,7 +50,7 @@ import { defineComponent } from 'vue'
 import { useCff } from '../store/cff'
 
 export default defineComponent({
-    name: 'Start',
+    name: 'PageStart',
```

Comment:
For consistency, this should be 

```suggestion
    name: 'Start',
```